open_academy/models/open_academy_session.py

```diff
@@ -1,14 +1,59 @@
-from odoo import models, fields
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
 
 
 class OpenAcademySession(models.Model):
     _name = "open.academy.session"
     _description = "model of session on open academy"
 
     name = fields.Char(required=True, size=50)
-    duration = fields.Text()
-    start_date = fields.Datetime()
-    number_seats = fields.Integer()
+    duration = fields.Integer()
+    active = fields.Boolean(default=True)
+    start_date = fields.Datetime(default=fields.Date.context_today)
+    end_date = fields.Datetime()
+    number_seats = fields.Integer(required=True)
+    number_seats_taken = fields.Integer(compute="_compute_number_seats_available")
     instructor_id = fields.Many2one("res.partner", "instructor")
     course_id = fields.Many2one("open.academy.course", required=True)
     attendees_ids = fields.Many2many("res.partner", "session_res_partner_rel")
```

Comment:
```suggestion
   attendees_count = fields.Integer(compute="_compute_number_taken_seats")
```

Reply:
In this case i can't change this field due to this is a field that make a relation between models and its necessary to someone later exercises

Reply:
I don't want you to change or remove this field, I want you create a new field `attendees_count`.